core/src/test/java/io/grpc/util/RoundRobinLoadBalancerTest.java

```diff
@@ -360,6 +377,25 @@ private Subchannel createMockSubchannel() {
     return subchannel;
   }
 
+  private void mockSubchannelAttributes(Subchannel subchannel, Attributes attributes) {
+    when(subchannel.getAttributes()).thenReturn(attributes);
```

Comment:
This is the proper way to mock the Subchannel. Please delete `createMockSubchannel()` and replace its call sites with a plain `mock(Subchannel.class)` (without stubbing `getAttributes()`).

Because this is a one-liner with only one caller (after deleting `mockSubchannelState` as I suggest below), it's better to merge it into the call site.